aiida/cmdline/commands/cmd_calcjob.py

```diff
@@ -267,8 +279,9 @@ def calcjob_cleanworkdir(calcjobs, past_days, older_than, computers, force):
         transport = orm.AuthInfo.objects.get(dbcomputer_id=computer.id, aiidauser_id=user.id).get_transport()
 
         with transport:
-            for path in paths:
+            for (calcjob_id, path) in paths:
                 clean_remote(transport, path)
                 counter += 1
+                load_node(calcjob_id).set_extra('workdir_cleaned', True)
```

Comment:
if you are going to load the node again, wouldn't it make more sense to have `get_calcjob_remote_paths` have the argument `include_calcjob` and return the node. Since you get it from the query, this will be a lot more efficient.